WordPress/Learn
# Migrate `wporg_lesson_category` taxonomy usage to `topic`

See #1037 

### Areas affected

1. Lesson Plan landing page Topic tiles display `topic` terms now, not `wporg_lesson_category`
3. The Lesson Plan Category taxonomy is no longer registered

### How to test
1. Ensure there is no Lesson Plan Category taxonomy
3. View the Lesson Plan landing page at `/lesson-plans/` and ensure the correct sticky terms are displayed with the correct icon, and the General topic is first
